src/card-product.ts

```diff
@@ -12,7 +12,11 @@ import {
   ImagesCarrier,
   ExpirationOffset,
 } from './card'
-import { MarqetaError } from './'
+import {
+  cleanMetatData,
```

Comment:
I think this is oddly spelled... did you mean `cleanMetaData()`? Because you have an extra `t` in it...